src/ReactiveDomain/ReactiveDomain/Memory/WrappedSquareImage.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using ReactiveDomain.FrameFormats;
+
+namespace ReactiveDomain.Memory
+{
+    public class WrappedSquareImage : WrappedBuffer
+    {
+        public unsafe WrappedSquareImage(BufferManager manager, PinnedBuffer buffer, int dimension, int bytesPerPixel) :
```

Comment:
Cosmetic (esp. since this is public code): clean up formatting to match param conventions, switch indentation, unnecessary braces, etc.